app/views/cloud_volume/backup_select.html.haml

```diff
@@ -1,4 +1,8 @@
-%form#form_div{:name => "angularForm", 'ng-controller' => "cloudVolumeFormController"}
+%form#form_div{:name => "angularForm",
```

Comment:
It's used for OpenStack provider: `Restore from a backup of this Cloud Volume`. I am unable to actually create a backup on our OpenStack testbed, so I could only ensure that the functions are being called exactly the same as they were called before.